views/v_ksm_internal_marketsheet.sql

```diff
@@ -472,14 +473,14 @@ INNER JOIN PROSPECT_ENTITY PE
 ON PHF.prospect_id = PE.PROSPECT_ID
   AND PE.PRIMARY_IND = 'Y'
   --- Kellogg Proposals Only + Active Proposals! 
-WHERE PHF.ksm_proposal_ind = 'Y'
+WHERE (PHF.ksm_proposal_ind = 'Y'
 and PHF.proposal_active_calc = 'Active'
-AND (PHF.PROPOSAL_STATUS_CODE IN ('A', 'C', '5', '7', '8') --- Just active proposals where status = 'A' /*, 'C', '5', '7', '8')*/ -- anticipated/submitted/approved/declined/funded
+AND (PHF.PROPOSAL_STATUS_CODE IN ('A','C','5')---, 'C', '5', '7', '8') --- Just anticipated proposals where status = 'A' /*, 'C', '5', '7', '8')*/ -- anticipated/submitted/approved/declined/funded
   AND PHF.ask_date BETWEEN rpt_pbh634.ksm_pkg_tmp.to_date2('9/01/2022','MM-DD-YY')
           AND rpt_pbh634.ksm_pkg_tmp.to_date2('8/31/2023','MM-DD-YY')) OR
-      (PHF.PROPOSAL_STATUS_CODE IN ('A', 'C', '5', '7', '8') -- anticipated/submitted/approved/declined/funded
+      (PHF.PROPOSAL_STATUS_CODE IN ('A','C','5')---, 'C', '5', '7', '8') -- anticipated/submitted/approved/declined/funded
   AND PHF.CLOSE_DATE BETWEEN rpt_pbh634.ksm_pkg_tmp.to_date2('9/01/2022','MM-DD-YY')
```

Comment:
You shouldn't need to hard code this - what about using v_current_calendar? prev_fy_start, curr_fy_start, next_fy_start